src/containers/MultidisciplinarySubject/MultidisciplinarySubjectArticlePage.jsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) 2020-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import Spinner from '@ndla/ui/lib/Spinner';
+import { useGraphQuery } from '../../util/runQueries';
+import { topicQueryWithPathTopics } from '../../queries';
+import { getUrnIdsFromProps } from '../../routeHelpers';
+import { MultidisciplinarySubjectArticle } from './components/MultidisciplinarySubjectArticle';
+
+const MultidisciplinarySubjectArticlePage = ({ match, locale }) => {
+  const { topicId } = getUrnIdsFromProps({ match });
+
+  const { data, loading } = useGraphQuery(topicQueryWithPathTopics, {
+    variables: { topicId },
+  });
+
+  const [pageUrl, setPageUrl] = useState('');
+  useEffect(() => {
+    setPageUrl(window.location);
```

Comment:
Kva skjer her ved SSR?

Reply:
vet ikke, har ikke endra på det i min branch

Reply:
ok.

Reply:
Dette krasha ved ssr så eg fiksa det sammen med at eg la på ekstra data for subject for at dimensions skulle vises.